index.js

```diff
@@ -92,7 +125,20 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+if(!movies.length){
+  throw "error";
+}
+
+
+let movieGenreArr = movies.filter((obj)=>{
+  if(obj.genre.toLowerCase() === genre.toLowerCase()){
```

Comment:
```suggestion
  if(obj.genre.toLowerCase().includes(genre.toLowerCase())){
```
You were really close with this one too! The reason you need to use includes here is because each movie object's genre had multiple genres listed, not just one.